Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemGroupsController.cs

```diff
@@ -131,11 +144,24 @@ private bool IsModelAndContestValid(ViewModelType model)
             if (string.IsNullOrWhiteSpace(model.ContestName))
             {
                 this.ModelState.AddModelError(nameof(model.ContestName), Resource.Contest_required);
+                return false;
             }
 
-            if (!this.contestsData.ExistsById(model.ContestId))
+            var contest = this.contestsData
+                .GetByIdQuery(model.ContestId)
+                .Select(c => new { c.Type })
+                .FirstOrDefault();
+
+            if (contest == null)
```

Comment:
I believe he wrapped the c.Type in an anonymous object, which would be null in this case.

Reply:
That's why I wrapped in an anonymous object, because I don't want to check if the type is 0, because the type might even be optional, I want to check first if such a contest exists, then to validate only if the type is different from `OnlinePracticalExam`